h2/settings.py

```diff
@@ -254,6 +254,13 @@ def __len__(self):
         return len(self._settings)
 
 
+    def __eq__(self,other):
+        return self._settings==other._settings
```

Comment:
Style nit: we need a space either side of the operator.

I should also note that this does not behave correctly when compared with objects that are not `Settings` objects: specifically, it'll probably throw `AttributeError`s.

To deal with this, you need to confirm that the other object is an instance of `Settings` before doing the comparison. If it's not, you want to indicate that we don't know how to compare them by returning `NotImplemented`.